### ISSUE TITLE
Find in node contents and names at the same time

### ISSUE THREAD

**Issue by golimarrrr** [giuspen/cherrytree#513]
Most times when you are searching inside a large notebook you just want to know where some word appears anywhere, so you end up searching both in Node Contents and then in Node Names.
Would it be better to integrate both search dialogs into one, with an option to find in Node Contents, Node Names, or both?

**Comment by giuspen** [comment]
yes it is a good point and I already thought about it, after the porting is complete I'll do it

**Comment by metal450** [comment]
As a new user, I thought the exact same thing (initially I actually thought it was a bug, as search wasn't returning data that I knew to be there - but I later realized I had to separately search with Ctrl+Shift+F and  then with Ctrl+T).

I agree that the best solution would be a single unified "search" dialog (Ctrl+Shift+F), which could just have checkboxes for what to include in the search: content, names, tags, etc :)

**Comment by giuspen** [comment]
Will be in the next version. I also unified the search in all nodes with the search in selected node and subnodes (all options as checkboxes in the search dialog)

**Comment by metal450** [comment]
Awesome, thanks! can't wait to try it out :)

**Comment by MJimitater** [comment]
Exciting to see that new features are being added, thanks @giuspen ! Seems like porting is then complete - thanks to the team and special shoutout to @txe !

**Comment by datavectors** [comment]
An added tip is to use ripgrep to search through a *,ctd file (or in my case many CT files scattered throughout my filesystem - Ubuntu)

https://github.com/BurntSushi/ripgrep

**Comment by metal450** [comment]
Thanks for this @giuspen! Just one small ask (I'll post it as a separate issue too, just wanted to follow-up here because it's related). Although the unified search dialog is super appreciated, I do think it's still useful to retain the Ctrl+T keyboard shortcut to search in names only, which seems gone in the current build.  That way, if you happen to *know* you're looking for a node name (which is much, much faster), you can still do it without needing to use the mouse to alter the checkboxes. 

So basically, Ctrl+Shift+F behaves as-now. And Ctrl+T brings up that same the 'unified search' dialog, but with only 'search in node names' selected.  Does that make sense? So that even with the new unified dialog, there's still a keyboard-only way to trigger 'search in node names', if desired for its speed.
